selenium/che-selenium-test/src/test/resources/suites/CheSuite.xml

```diff
@@ -287,6 +287,7 @@
                     <exclude name="pullConflictsTest"/>
                 </methods>
             </class>
+            <class name="org.eclipse.che.selenium.git.RevertCommitTest"/>
```

Comment:
Please, don't forget to add test to the [CodenvyOnpremSuite.xml](https://github.com/codenvy/codenvy/blob/master/selenium/codenvy-selenium-test/src/test/resources/suites/CodenvyOnpremSuite.xml) and [CodenvySaasSuite.xml](https://github.com/codenvy/saas/blob/master/selenium/saas-selenium-test/src/test/resources/suites/CodenvySaasSuite.xml) as well.